### ISSUE TITLE
Training Data

### ISSUE THREAD

**Issue by KasparPeterson** [aiiu-lab/I3D_Simple#1]
Hello,

Thanks for this work. I tried training the model on faceforensics dataset where faces were cropped out. However I cannot get the training/validation accuracy to go over 80%. That's why I have some questions:

1. Would it be possible to share the dataset used for training or describe the data preprocessing in README?
2. As I understand the accuracy on faceforensics/deepfakes dataset was 94.6% and on faceforensics/faceswap the result was 53.9%?
